### DISCUSSION TITLE
Beamer frame attributes using horizontal rules in Quarto

### DISCUSSION THREAD

**Discussion by DavidJesse21** [quarto-dev/quarto-cli#5451] [Q&A]
Hi,

I just stumbled across an issue regarding the usage of frame attributes for beamer output in combination with the usage of horizontal rules in Quarto (---).
Since I want to have a slide without a title, I used the horizontal rule.
At the same time I needed the *fragile* frame attribute.
For a slide with a title I would just type `## Slide tile {.fragile}`.
However, I am not sure how or if adding frame attributes works for slides created with a horizontal rule.
Apparently, `--- {.fragile}` does not work.
Does anyone have an idea, how I could make this work?

Cheers
David

**Comment by mcanouil** [comment]
Use an empty title.

````qmd
---
title: Demo
format: beamer
---

## {.fragile}

More stuff here
````
